docs/zkapps/tutorials/interacting-with-zkapps-server-side.mdx

```diff
@@ -12,7 +12,7 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
 :::note
 
-This tutorial was last tested with [SnarkyJS](https://www.npmjs.com/package/snarkyjs) 0.8.0.
+This tutorial was last tested with [o1js](https://www.npmjs.com/package/o1js) 0.12.1.
```

Comment:
if you have tested with 12.1, great! We have not officially tested with o1js, so the SnarkyJS release is accurate